packages/search-input/src/SearchInput/SearchInput.tsx

```diff
@@ -245,6 +251,9 @@ export const SearchInput = React.forwardRef<HTMLInputElement, SearchInputProps>(
 
     const handleInputChange: ChangeEventHandler<HTMLInputElement> = e => {
       handleChange?.(e);
+      __INTERNAL__inputProps__ &&
+        __INTERNAL__inputProps__.onChange &&
+        __INTERNAL__inputProps__.onChange(e);
```

Comment:
if you have `__INTERNAL__inputProps__.onChange` would `handleInputChange` ever be called since `__INTERNAL__inputProps__.onChange` overrides `onChange` on the input?